LearningX.md

```diff
@@ -6,3 +6,4 @@ This has useful links to learn X.
 4. [GitHub Help Guide](https://help.github.com/)
 5. [IRC](https://fedoramagazine.org/beginners-guide-irc/)
 6. [Git and Github in Plain English](https://red-badger.com/blog/2016/11/29/gitgithub-in-plain-english)
+7. [Git Beginner's Guide for Dummies](http://backlogtool.com/git-guide/en)
```

Comment:
Does this link lead you somewhere? I didn't find any git guide on this website, closing the pull request. Add another commit having the guide I'll merge.